.github/scripts/internal/enums/chains.go

```diff
@@ -0,0 +1,33 @@
+package enums
+
+type Chain string
+
+const (
+	Juno         Chain = "juno"
+	Kujira       Chain = "kujira"
+	Osmosis      Chain = "osmosis"
+	Terra        Chain = "terra"
+	TerraClassic Chain = "terraclassic"
+)
+
+func (c Chain) Name() string {
+	return string(c)
+}
+
+func getAllChains() []Chain {
+	return []Chain{
+		Juno,
+		Kujira,
+		Osmosis,
+		Terra,
+		TerraClassic,
+	}
+}
+
+func GetAllChainNames() []string {
```

Comment:
Not a big fan of maintaining a constant list of supported chains as that would mean the CI needs to be updated whenever a new chain is added. Could you make it so that it "automatically" works on all chains?